waspc/data/Generator/templates/server/src/auth/hooks.ts

```diff
@@ -0,0 +1,97 @@
+{{={= =}=}}
+import { prisma } from 'wasp/server'
+import type {
+  OnAfterOAuthTokenReceivedHookFn,
+  OnAfterSignupHookFn,
+  OnBeforeOAuthRedirectHookFn,
+  OnBeforeSignupHookFn,
+} from 'wasp/server/auth'
+{=# onBeforeSignupHook.isDefined =}
+{=& onBeforeSignupHook.importStatement =}
+{=/ onBeforeSignupHook.isDefined =}
+{=# onAfterSignupHook.isDefined =}
+{=& onAfterSignupHook.importStatement =}
+{=/ onAfterSignupHook.isDefined =}
+{=# onBeforeOAuthRedirectHook.isDefined =}
+{=& onBeforeOAuthRedirectHook.importStatement =}
+{=/ onBeforeOAuthRedirectHook.isDefined =}
+{=# onAfterOAuthTokenReceivedHook.isDefined =}
+{=& onAfterOAuthTokenReceivedHook.importStatement =}
+{=/ onAfterOAuthTokenReceivedHook.isDefined =}
+
+/*
```

Comment:
We wrap the user defined functions so we don't have to call them with common params like `prisma` or `hookName` when invoking the hooks in our server code.